### PR TITLE
Order inputs alphabetically

### PR DESCRIPTION
**Pull request by toy**
Order inputs alphabetically, to easily find relevant option and know where to insert a new one

### CODE REVIEW

**Review by javierjulio — APPROVED** [review]
Thanks! We do have many inputs so this should help. I'm good with the change. If no objections, then I'll merge.

**Comment by toy** [comment]
@javierjulio Truing to nudge about this PR and about #103

File: README.md [hunk 1]
```diff
@@ -26,17 +26,35 @@ With `reporter: github-pr-review` a comment is added to the Pull Request Convers
 
 ## Inputs
 
+<!-- Please add inputs maintaining alphabetical order -->
```
Comment (javierjulio) [inline README.md:29]:
```suggestion
<!-- Please maintain inputs in alphabetical order -->
```

File: action.yml [hunk 2]
```diff
@@ -2,20 +2,20 @@ name: 'Run rubocop with reviewdog'
 description: '🐶 Run rubocop with reviewdog on pull requests to improve code review experience.'
 author: 'mgrachev (reviewdog)'
 inputs:
+  # Please add inputs maintaining alphabetical order
```
Comment (javierjulio) [inline action.yml:5]:
```suggestion
  # Please maintain inputs in alphabetical order
```

File: action.yml [hunk 3]
```diff
@@ -55,19 +56,20 @@ runs:
         REVIEWDOG_VERSION: v0.17.4
         # INPUT_<VARIABLE_NAME> is not available in Composite run steps
         # https://github.community/t/input-variable-name-is-not-available-in-composite-run-steps/127611
+        # Please add inputs maintaining alphabetical order
```
Comment (javierjulio) [inline action.yml:59]:
```suggestion
        # Please maintain inputs in alphabetical order
```

**Review by javierjulio — APPROVED** [review]
Thanks!

**Comment by review-dog** [comment]
Hi, @toy! We merged your PR to reviewdog! 🐶
Thank you for your contribution! ✨

**We just invited you to join the @reviewdog organization on GitHub.**
Accept the invite by visiting https://github.com/orgs/reviewdog/invitation.
By joining the team, you'll be a part of reviewdog community and can help the maintenance of reviewdog.

Thanks again!

**Pull request merged**